package.json

```diff
@@ -8,6 +8,7 @@
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
     "test:unit": "vue-cli-service test:unit",
+    "test:clear:snapshot": "npm run test:unit -- -i -u",
```

Comment:
When I run this command with `yarn`, I get an error `.... Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.`. Can you resolve it?